scripts/ty_benchmark/README.md

```diff
@@ -7,15 +7,24 @@
 
 1. Build ty: `cargo build --bin ty --release`
 1. `cd` into the benchmark directory: `cd scripts/ty_benchmark`
+1. Install Pyright: `npm install`
 1. Run benchmarks: `uv run benchmark`
 
+Requires hyperfine 1.20 or newer.
+
 ## Known limitations
 
-ty only implements a tiny fraction of Mypy's and Pyright's functionality,
-so the benchmarks aren't in any way a fair comparison today. However,
-they'll become more meaningful as we build out more type checking features in ty.
+The tested type checkers implement Python's type system to varying degrees and
+some projects only successfully pass type checking using a specific type checker.
 
 ### Windows support
 
 The script should work on Windows, but we haven't tested it yet.
 We do make use of `shlex` which has known limitations when using non-POSIX shells.
+
+## Updating the benchmark
+
+The benchmark script supports snapshoting the results when running with `--snapshot` and `--accept`.
+The goal of those snapshots is to catch accidental regressions. They are not intended
```

Comment:
What would be an example of such a regression? A case where one type checker suddenly outputs WAY more/fewer diagnostics?